proto/result.go

```diff
@@ -0,0 +1,434 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package proto
+
+import (
+	"fmt"
+
+	timestamppb "google.golang.org/protobuf/types/known/timestamppb"
+)
+
+// Claim values as defined by
+// https://datatracker.ietf.org/doc/draft-ietf-rats-ar4si/
+const (
```

Comment:
this is awesome, and it's key information for plugin writers.  I am wondering how to make it as visible as possible?